src/main/java/uk/gov/ons/ctp/response/collection/exercise/message/CollectionInstrumentLoadedReceiver.java

```diff
@@ -21,8 +20,8 @@
   public void acceptSampleUnit(CollectionInstrumentMessageDTO message) throws CTPException {
     log.with(message).debug("Consumed message");
 
-    UUID collexId = message.getExerciseId();
+    // UUID collexId = message.getExerciseId();
 
```

Comment:
Do we need this class anymore if all it does is log?

Reply:
We don't will take it out.